src/devices/sound/gt155.h

```diff
@@ -0,0 +1,104 @@
+// license:BSD-3-Clause
+// copyright-holders: Devin Acker
+/***************************************************************************
+    Casio GT913 sound (HLE)
+***************************************************************************/
+
+#ifndef MAME_DEVICES_SOUND_GT155_H
+#define MAME_DEVICES_SOUND_GT155_H
```

Comment:
We don’t put `DEVICES_` in `#include` guards since bad thing happen if things shadow each other in src/devices and src/mame anyway – it should be `MAME_SOUND_GT155_H`.